### PR TITLE
Connected email service and added first email template

### LINKED ISSUE #141: Email templates
Create a service (or something like that) that can generate html emails from a few parameters. For now, we should focus on one template: the email that is sent when a consultation request is fulfilled. Later on, we will need more templates, so the solution should be expandable. The frame of the email template can be found in [our Drive](https://drive.google.com/drive/u/1/folders/132VETsQSV9aXhlz8fYIRSzRa1LK-HUSX), might need some modifications

### PR DESCRIPTION
**Pull request by Tschonti**
I connected our brand new email ~micro~service using https://github.com/kir-dev/email-template

I disabled email sending in non-production, so if you want to test it, set NODE_ENV to production in `.env`. I wanted to put all the email-sending functions (for now, there's only one) in one place, so I used the `mailing.service.ts` for this as well. These could potentially be called from all sorts of places, so decided to use nest's event emitter system for calling these methods. Not sure if it's the right thing to use here, but I just learnt about it while reading last year's Nest courses by Trisz.

And I created the first email template, sent when a consultation fulfils a request (to those that requested or supported it).
image

I tested it in local and docker, works fine
Closes #141, #142

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -13,3 +13,6 @@ BACKEND_PORT=3300
 NODE_ENV=development
 FRONTEND_HOST=http://localhost:3000
 FRONTEND_AUTHORIZED_URL=http://localhost:3000/authorized
+
+MAIL_SERVER_URL=https://mail.kir-dev.hu
```
Comment (berenteb) [inline .env.example:17]: Remove this pls, we dont want to expose the service.

File: Dockerfile [hunk 2]
```diff
@@ -56,5 +56,6 @@ COPY --from=build /usr/src/app/node_modules ./node_modules
 COPY --from=build /usr/src/app/dist ./dist
 COPY --from=build /usr/src/app/package.json ./
 COPY --from=build /usr/src/app/prisma ./prisma
+COPY --from=build /usr/src/app/src/mailing/templates ./src/mailing/templates
```
Comment (berenteb) [inline Dockerfile:59]: Okay if this works but would be better to have the path in env var so you are not mixing folder structure in Docker

**Review by berenteb — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (Tschonti) [reply .env.example:17]: 😶‍🌫️

**Review by DannySS123 — APPROVED** [review]
A megfogalmazáson még lehetne csiszolni, de nekem nincs ötletem.
Amúgy fasza lett :D

**Review by DannySS123 — APPROVED** [review]
Looks good 🆗

**Pull request merged**